redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -172,10 +172,14 @@ protected function getImageSource()
     {
         ob_start();
         if ($this->format == 'jpg' || $this->format == 'jpeg') {
-            $this->image['quality'] = rex_config::get('media_manager', 'jpg_quality', 85);
-            imagejpeg($this->image['src'], null, $this->image['quality']);
+            if (null !== $progressive = $this->getImageProperty('jpg_progressive')) {
+                imageinterlace($this->image['src'], $progressive ? 1 : 0);
+            }
+            $quality = $this->getImageProperty('jpg_quality', rex_config::get('media_manager', 'jpg_quality', 85));
+            imagejpeg($this->image['src'], null, $quality);
         } elseif ($this->format == 'png') {
-            imagepng($this->image['src']);
+            $compression = $this->getImageProperty('png_compression', -1);
+            imagepng($this->image['src'], null, $compression);
         } elseif ($this->format == 'gif') {
             imagegif($this->image['src']);
         } elseif ($this->format == 'wbmp') {
```

Comment:
Dann müsste man aber im Nachgang wieder diese Dateien bearbeiten, deswegen dachte ich, in einem Rutsch abarbeiten (außer es gibt noch keine Entscheidung zu webp, dann hab ich nichts gesagt)